packages/app/obojobo-repository/server/routes/library.js

```diff
@@ -39,10 +40,58 @@ router.route('/library/module-icon/:moduleId').get((req, res) => {
 		return
 	}
 
-	const pattern = GeoPattern.generate(req.params.moduleId)
+	// array of ColorBrewer sequences
+	const colors = [
+		'Blues',
+		'Greens',
+		'Oranges',
+		'Purples',
+		'Reds',
+		'BuGn',
+		'BuPu',
+		'GnBu',
+		'OrRd',
+		'PuBuGn',
+		'PuBu',
+		'PuRd',
+		'RdPu',
+		'YlGnBu',
+		'YlGn',
+		'YlOrBr',
+		'YlOrRd'
+	]
+	// deterministically choose a 'random' number to select a color scheme
+	const deterministicRandom = seedrandom(req.params.moduleId)()
+	const color = colors[Math.floor(deterministicRandom * colors.length)]
+
+	const pattern = trianglify({
+		width: 200,
+		height: 200,
+		cellSize: 65,
+		variance: 0.8,
+		xColors: color,
+		strokeWidth: 0.5,
+		seed: req.params.moduleId
+	})
+
 	res.setHeader('ETag', req.params.moduleId)
 	res.setHeader('Content-Type', 'image/svg+xml')
-	res.send(pattern.toString())
+
+	// build the svg
+	const svg = `<svg viewBox="0 0 130 150" xmlns="http://www.w3.org/2000/svg">
+		<mask id="hexagon-mask">
+			<path
+				fill="white"
+				d="M56.29165124598851 4.999999999999999Q64.9519052838329 0 73.61215932167728 4.999999999999999L121.24355652982142 32.5Q129.9038105676658 37.5 129.9038105676658 47.5L129.9038105676658 102.5Q129.9038105676658 112.5 121.24355652982142 117.5L73.61215932167728 145Q64.9519052838329 150 56.29165124598851 145L8.660254037844387 117.5Q0 112.5 0 102.5L0 47.5Q0 37.5 8.660254037844387 32.5Z"
+			/>
+		</mask>
+
+		<g mask="url(#hexagon-mask)">
+			${pattern.svg().innerHTML}
```

Comment:
It seems the API changed, this works:

```suggestion
			${pattern.toSVG()}
```

Reply:
Perplexing 🤔  I took the above screenshot w/ it functioning.